sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -57,8 +57,8 @@ public class SensitivityAnalysisResult {
     private final Map<String, SensitivityContingencyStatus> statusByContingencyId = new HashMap<>();
 
     public enum Status {
-        CONVERGED,
-        FAILED,
+        SUCCEED,
+        FAIL,
```

Comment:
It is more common to have `SUCCESS/FAILURE` messages or at least  `SUCCESS/FAILED`

Reply:
For a status an adjective seems more suitable to me : Succesful / Failing.
No strong feeling about it, from @annetill the issue was simply that converged was not explicit enough.